VerticalSlice2D/Assets/Scripts/HeartMovement.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HeartMovement : MonoBehaviour
+{
+    public SpriteRenderer box;
```

Comment:
[SerializeField] toevoegen voor variabelen die je alleen in de inspector wil aanpassen.

Reply:
Ik heb het aangepast